### PR TITLE
✨ Admin panel

### PR DESCRIPTION
**Pull request by samir-0711**

### CODE REVIEW

**Review by patheticGeek — CHANGES_REQUESTED** [review]
Try doing the things i mentioned, if you need help ping me

File: src/utils/api.ts [hunk 1]
```diff
@@ -24,7 +24,9 @@ export const API_ENDPOINTS = {
   WEEKLY_TODO: `/api/v1/weekly-todo`,
   ONBOARDING: `/api/v1/users/onboard`,
   REPORT_AN_ISSUE: `api/v1/internal-feedback`,
-  ADMIN_PANEL: `api/v1/admin/`,
+  ADMIN_PANEL_USERS: `api/v1/admin/users`,
+  ADMIN_PANEL_ONBOARD: `api/v1/admin/users?filter[is_discord_form_filled]=true`,
```
[the remark this answers is not in the document]
Comment (samir-0711) [reply src/utils/api.ts]: You are reviewing outdated code.

File: src/services/admin.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    pageIndex,
+    pageLinks,
+    data
+  }
+}
+
+export const useAdmin = (
+  url: string | undefined,
+  key: string | undefined,
+  page_number: number
```
[the remark this answers is not in the document]
Comment (samir-0711) [reply src/services/admin.ts]: You are reviewing outdated code

File: src/services/admin.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import { useQuery } from 'react-query'
+
+import axios from 'config/axios'
+
+const fetchAdminData = async (url: string | undefined, page_number: number) => {
+  if (!url) {
+    return null
+  }
+
+  console.log(url)
```
[the remark this answers is not in the document]
Comment (samir-0711) [reply src/services/admin.ts]: You are reviewing outdated code

File: src/pages/Admin/Admin.tsx [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+    total: 1
+  })
+  const [pageLinks, setPageLinks] = useState({})
+
+  const [selectedOption, setSelectedOption] = useState<OptionType>(
+    optionsMap[0]
+  )
+
+  const handleOption = (key: string) => {
+    console.log(key)
```
[the remark this answers is not in the document]
Comment (samir-0711) [reply src/pages/Admin/Admin.tsx]: You are reviewing outdated code

**Review by sanjay270899 — APPROVED** [review]

**Review by patheticGeek — APPROVED** [review]
LGTM

**Pull request merged**